sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Generated/Models/AlertIntent.cs

```diff
@@ -1,36 +0,0 @@
-// <auto-generated>
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for
-// license information.
-//
-// Code generated by Microsoft (R) AutoRest Code Generator.
-// Changes may cause incorrect behavior and will be lost if the code is
-// regenerated.
-// </auto-generated>
-
-namespace Microsoft.Azure.Management.Security.Models
-{
-
-    /// <summary>
-    /// Defines values for AlertIntent.
-    /// </summary>
-    public static class AlertIntent
```

Comment:
I wonder why this was removed

Reply:
This is related to Iot alerts which they deprecated in this swagger PR - https://github.com/Azure/azure-rest-api-specs/pull/15434/files